_commands/basics/alias.md

```diff
@@ -0,0 +1,106 @@
+---
+---
+
+alias
+-------
+
+`alias` is a command to create shortcut to commands and applications in the terminal.
+
+~~~bash
+$ alias l="ls"
+$ l
+Documents  Downloads  Music     Public     Videos
+Desktop    Pictures   Templates
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### Usage
+
+Things to note: when using alias, make sure that there is no space around the equal sign.
+
+~~~bash
+$ alias l = "ls"
+alias l='ls -CF'
+bash: alias: =: not found
+~~~
+
+Another thing to note is that aliases would overwrite existing commands. For example:
+
+~~~bash
+$ alias ls="git status"
+$ ls
+On branch master
+Your branch is up-to-date with 'origin/master'.
+~~~
+
+Also, aliases are temporary to the current shell. To make the aliases permanent, please refer to the section below "Making aliases permanent".
+
+#### Example
+
+The above examples do not make much sense, since they simplify a two character command. The following examples show how alias can save you a lot of time.
+
+~~~bash
+$ alias s="git status"
+$ s
+On branch master
+Your branch is up-to-date with 'origin/master'.
+~~~
+
+This shows that you can alias commands with multiple words.
+
+#### Example
+
+~~~bash
+$ alias gdesk="cd ~/Desktop"
+$ pwd
+/home/user/Desktop
```

Comment:
Creating an alias would not actually run the command – could you clarify this example?